message/messaging_service.hh

```diff
@@ -182,7 +182,8 @@ enum class messaging_verb : int32_t {
     DIRECT_FD_PING = 63,
     RAFT_TOPOLOGY_CMD = 64,
     RAFT_PULL_TOPOLOGY_SNAPSHOT = 65,
-    LAST = 66,
+    RAFT_STREAM_TABLET = 66,
```

Comment:
Not related to raft except incidentally. Suggest to open a  new prefix TABLET_ for tablet operations.

Reply:
Will change to TABLET_STREAM_DATA